routes/RESTful_API_Docs.md

```diff
@@ -274,33 +306,37 @@ req.body = {
 
 ---
 
-# Plants Routes 
+# Plants Routes
+
 **Features**
-  - Only for authenticated and authorized users.
-  - Plant CRUD Operations
-  - Query Plant in Trefle
+
+- Only for authenticated and authorized users.
+- Plant CRUD Operations
+- Query Plant in Trefle
 
 ### Plants Schema
-|    Key    |    Column     |    Type    |
-|    :---:    |    :---:     |    :---:     |
-| PK | id | serial unique | 
-|  | garden_id | smallint | 
-|  | user_id | smallint | 
-|  | common_name | varchar(255) |
-|  | scientific_name | varchar(255) |
-|  | trefle_id | smallint | 
-|  | duration | varchar(255) |
-|  | outdoor_plant | boolean |
-|  | last_watered | date |
-|  | images | json |
-|  | foliage | json |
-|  | fruit_or_seed | json |
-|  | growth | json |
-|  | seed | json |
-|  | specifications | json |
-|  | family_common_name | varchar(255) |
+
+| Key |       Column       |     Type      |
+| :-: | :----------------: | :-----------: |
```

Comment:
"name" (user specific name), should be added to this now?

Reply:
good catch!